docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,119 @@
+(CSMR) by the risk-weighted LTBI prevalence in order to model TB among at risk
+populatoin. GBD ran a mixed effects regression (with region random effects)
+using logit transformed MI ratios from locations with a 4-star or above rating
+(countries with poor surveillance systems are classified as less than 4 stars)
+on causes of death with HAQI as a covariate anchoring the lower bound of the
+HAQI scale with a data point reported in the Bangalore study to predict age-sex
+specific MI ratios for all locations and years. Then the age-sex-specific
+incidence were estimated based on those predicted MI ratios. GBD used DisMod-MR
+2.1 (Bayesian meta-regression tool) to generate consistent trends in all
+parameters, and multiplied the outputs by the risk-weighted prevalence of LTBI
+to get population-level estimates of incidence and prevalence. GBD disaggregated
+the all-form TB outputs from DisMod-MR 2.1 into MDR-TB and XDR-TB by HIV status.
+To do so, they estimated the proportions of TB cases with MDR-TB for all
+locations and years, and then estimated the proportions of MDR-TB among
+HIV-negative and HIV-positive individuals based on the risk of MDR-TB associated
+with HIV infection from a meta-analysis. To split MDR-TB into MDR-TB with and
+without extensive drug resistance, GBD pooled the notification and survey data
+on the proportion of MDR-TB cases with extensive drug resistance by
+super-region, and applied these proportions to MDR-TB cases among HIV-negative
+and HIV-positive individuals, respectively. [GBD-2017-YLD-Capstone-Appendix-1]_
+
+Summary model description
++++++++++++++++++++++++++
+It's a complex joint model that includes both tuberculosis and HIV/AIDS. We
+model TB and HIV together because HIV has impacts on both progression rate of
+LTBI to active TB and active TB excess mortality rate. Note that we have
+excluded the active TB drug resistance and GBD predicted the proportion of new
+TB cases with MDR-TB or XDR-TB by ST-GPR. In our model, each simulant occupies
+a state at each discrete time step. It is convenient to distinguish two
+sub-state types: a healthy (susceptible) state and a disease (with condition)
+state. Many simulants began the simulation in a susceptible state with, for
+example, no TB and remain in the susceptible state until they contract LTBI,
+progress to active TB, or die due to TB or other causes. Each state is mutually
+exclusive (e.g. a simulant can be susceptible or be a prevalent case of TB, but
+not both). However, a simulant may have TB and still be susceptible to HIV or
+vice versa. Prevalence is the measure that describes the proportion of individuals
+with disease in the total population during the given simulation time step. The
+prevalence of a disease state determines what proportion of the population will
+occupy that disease state during each time step. Each disease state has transition
+attributes defining the simulant transitions into and out of the state. We
```

Comment:
This part also seems like a general attribute of vivarium models, and perhaps doesn't need to be specified here (though if there are nuances to the transition rates in this model, that makes sense to me to include/clarfiy!)